cmd/edgectl/aes_install.go

```diff
@@ -80,6 +85,16 @@ func aesInstall(cmd *cobra.Command, args []string) error {
 	kcontext, _ := cmd.Flags().GetString("context")
 	i := NewInstaller(verbose)
 
+	// If Scout is disabled (environment variable set to non-null), inform the user.
+	// TODO: should be i.scout.Disabled() when Alex's PR #2496 is merged.
+	if os.Getenv("SCOUT_DISABLE") != "" {
+		i.show.Printf(color.Notice.Sprintf("[aesInstall] Scout is disabled; metrics will not be written to Metriton."))
```

Comment:
This message is not going to be clear to the user (in the odd case that they use it) and thus it will cause them to be confused. Something like: "[aesInstall] Info: phone-home metrics have been disabled by environment variable." Why is this better? (a) it says "Info" so that users know that they can ignore it; (b) it explains a little bit about what is going on in language the user probably understands rather than Scout and Metriton".

Reply:
thanks, that's helpful.  I was under the impression this was a "datawire only" message so it is clear that this is better.

Reply:
Fixed in new push.